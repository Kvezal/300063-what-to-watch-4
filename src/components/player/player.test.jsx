import React from "react";
import render from "react-test-renderer";
import Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import Player from "./player";


Enzyme.configure({
  adapter: new Adapter(),
});

describe(`PlayerComponent`, () => {
  test(`should render component`, () => {
    const tree = render
      .create(
          <Player>
            <video/>
          </Player>,
          {createNodeMock: () => ({})}
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
