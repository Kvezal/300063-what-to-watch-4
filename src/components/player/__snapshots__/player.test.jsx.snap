// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerComponent should render component 1`] = `<video />`;
